app-crypt/libsecret/libsecret-0.20.5.ebuild

```diff
@@ -47,10 +47,28 @@ BDEPEND="
 			dev-python/dbus-python[${PYTHON_USEDEP}]
 			introspection? ( dev-python/pygobject:3[${PYTHON_USEDEP}] )')
 		introspection? ( >=dev-libs/gjs-1.32 )
+		tpm? (	app-crypt/swtpm
+			app-crypt/tpm2-abrmd
+			>=app-crypt/tpm2-tss-3.2.0 )
 	)
 	vala? ( $(vala_depend) )
 "
 
+tpm2_run_with_emulator() {
+	eval `dbus-launch --sh-syntax`
+	export XDG_CONFIG_HOME=${WORKDIR}/.config/swtpm
+	/usr/share/swtpm/swtpm-create-user-config-files
```

Comment:
BROOT or ESYSROOT, I guess the latter based on DEPEND, not that it really matters for tests anyway if needed in both